### PR TITLE
feat: add fluentbit config for splunk and apply it to node

### PR DESCRIPTION
**Pull request by trev-dev**
Jira: OFMCC-641

### CODE REVIEW

File: tools/openshift/backend.dc.yaml [hunk 1]
```diff
@@ -78,12 +72,56 @@ objects:
                 limits:
                   cpu: '${MAX_CPU}'
                   memory: '${MAX_MEM}'
+              volumeMounts:
```
Comment (weskubo-cgi) [inline tools/openshift/backend.dc.yaml:75]: This volumeMounts element isn't the same position as backend-tls.dc.yaml which could be confusing if we have to compare or make change to both files.

File: tools/openshift/backend.dc.yaml [hunk 2]
```diff
@@ -78,12 +72,56 @@ objects:
...
+                # - name: tls-certs
```
Comment (weskubo-cgi) [inline tools/openshift/backend.dc.yaml:76]: Can we just remove tls-certs? They won't ever be needed as we have the -tls specific template.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: tools/openshift/backend.dc.yaml (hunk 1 above)
Comment (trev-dev) [reply tools/openshift/backend.dc.yaml:75]: Moved to the bottom of the image config in both files

File: tools/openshift/backend.dc.yaml (hunk 2 above)
Comment (trev-dev) [reply tools/openshift/backend.dc.yaml]: Yes :D

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**